refactor(design): add footer to app layout

golddydev opened the pull request:
## Changes

* Move landing footer components to `src/components/`
* Split footer components into `FooterSections` and `FooterImage`
* Add `FooterSections` to app layout
* Refine app layout
* Remove app header's theme toggle (because that is in footer sections)
* Add scroll (horizontal, vertical) to job table header and content

## Related issues

This PR will close #368

mrgrauel reviewed (changes requested):
The footer isn't displayed in the agents gallery. Please make sure that the footer is added on every side in the app context.

golddydev commented:
> The footer isn't displayed in the agents gallery. Please make sure that the footer is added on every side in the app context.

Oh, fixed by [916a3dd](https://github.com/masumi-network/sokosumi/pull/487/commits/916a3ddb5043d25274555f22b692510aaae11c44)

## Why it didn't work?

It is because class names' issue (I updated app layout a bit)
Now it is all working good
